charts/nginx/values.yaml

```diff
@@ -124,3 +124,10 @@ defaultBackend:
   resources: {}
   readinessProbe: {}
   livenessProbe: {}
+
+cspPolicy:
```

Comment:
```suggestion
cdnPolicy:
```

Reply:
this change comes under csp policy

Reply:
then let's change `enabled` to `cdnEnabled`

Reply:
`cspPolicy.cdnEnabled` looks more legible